docs/references/ic-interface-spec.md

```diff
@@ -2273,7 +2273,25 @@ Indicates various information about the canister. It contains:
 
 -   A SHA256 hash of the module installed on the canister. This is `null` if the canister is empty.
 
--   The actual memory usage of the canister.
+-   The total memory usage of the canister, which includes all types of memory consumed by the canister.
+
+-   Detailed breakdown of memory usage into individual components (in bytes):
```

Comment:
High level comment: the interface spec uses the term "system state" mostly to refer to the system state tree which is different than what we call "system state" in the current implementation of the Internet Computer. Therefore, I would not necessarily try to capture this in the below but rather stay on a higher level describing what type of memory the field represents.

Additionally, for concepts that are covered elsewhere in the spec, we do not need to provide further clarifications imo. E.g. there's no need to explain what is contained in the canister history as the relevant section covers it.